client-admin/src/app/shared/components/event-calendar/event-calendar.component.html

```diff
@@ -18,7 +18,8 @@
           [class.selectedRange]="isSelectedRange(day)"
           [class.empty]="day.properties.empty"
         >
-          {{ day.date.day }}. {{ day.date.month }}.
+          <span *ngIf="!day.properties.today">{{ day.date.day }}. {{ day.date.month }}.</span>
+          <span *ngIf="day.properties.today"><ion-badge>{{ day.date.day }}. {{ day.date.month }}.</ion-badge></span>
```

Comment:
Mají nějaký konkrétní smysl ty spany?

Reply:
K ničemu, hodím tam ng-template s if else syntaxí, takto mi jen zjednodušovalo zápis (vlastně jsem to jen zkopíroval z toho svátku hned pod tím, ale u něj to má smysl kvůli float-right)

Reply:
můžeš i jen ng-container

Reply:
ale je to jedno